docs/docs/PythonUserGuide/install-without-pip.md

```diff
@@ -5,12 +5,12 @@
 2. You can download the BigDL release and nightly build from the [Release Page](../release-download.md)
```

Comment:
Add a note that only python 2.7 and 3.5 are currently supported.

Reply:
Added and highlighted.